src/mctp/mctp.h

```diff
@@ -31,25 +31,25 @@
 #define MCTP_BASELINE_TRAN_UNIT_SIZE    (64)
 
 #define MCTP_TRACE_FILTER (0 \
-        | LSHIFT(ERROR) \
-        | LSHIFT(WARN) \
-        | LSHIFT(DEBUG) \
-        | LSHIFT(INFO) \
-        | LSHIFT(INIT) \
+        | BITLSHIFT(1, ERROR) \
+        | BITLSHIFT(1, WARN) \
+        | BITLSHIFT(1, DEBUG) \
+        | BITLSHIFT(1, INFO) \
+        | BITLSHIFT(1, INIT) \
         )
 
 #if 1
 extern const char *mctp_trace_header[];

```

Comment:
Remove unnecessary code/comment.

Reply:
Since different C files use mctp_trace, we cannot remove mctp_trace_header here.

Reply:
Since different C files use mctp_trace, we cannot remove mctp_trace_header here.